spec/Section 6 -- Execution.md

```diff
@@ -4,18 +4,21 @@ A GraphQL service generates a response from a request via execution.
 
 :: A _request_ for execution consists of a few pieces of information:
 
-- The schema to use, typically solely provided by the GraphQL service.
-- A {Document} which must contain GraphQL {OperationDefinition} and may contain
-  {FragmentDefinition}.
-- Optionally: The name of the Operation in the Document to execute.
-- Optionally: Values for any Variables defined by the Operation.
-- An initial value corresponding to the root type being executed. Conceptually,
-  an initial value represents the "universe" of data available via a GraphQL
-  Service. It is common for a GraphQL Service to always use the same initial
-  value for every request.
-
-Given this information, the result of {ExecuteRequest()} produces the response,
-to be formatted according to the Response section below.
+- {schema}: The schema to use, typically solely provided by the GraphQL service.
```

Comment:
nit and not-this-PR: `GraphQL service` is mildly ambiguous, it's the execution service or the executor or something like that?

Reply:
Also nit: `GraphQL Service` capitalized like below?

Reply:
We use this term across the spec I believe